packages/binding-wotfirestore-browser-bundle/package.json

```diff
@@ -21,5 +24,10 @@
   },
   "scripts": {
     "build": "browserify -r vm:vm2 index.js --external coffee-script -o dist/binding-wotfirestore-bundle.js"
-  }
+  },
+  "bugs": {
+    "url": "https://github.com/hidetak/thingweb.node-wot/issues"
+  },
+  "homepage": "https://github.com/hidetak/thingweb.node-wot#readme",
+  "keywords": []
```

Comment:
Similar question, shouldn't it point to

https://github.com/eclipse/thingweb.node-wot/tree/master/packages/binding-wotfirestore